### PR TITLE
Update OCR model

### PR DESCRIPTION
**Pull request by qingqing01**

### CODE REVIEW

File: examples/ocr/seq2seq_attn.py [hunk 1]
```diff
@@ -249,7 +247,7 @@ def forward(self, target, initial_states, encoder_vec, encoder_proj):
         return pred
 
 
-class Seq2SeqAttModel(Model):
+class Seq2SeqAttModel(fluid.dygraph.Layer):
```
Comment (saxon-zh) [inline examples/ocr/seq2seq_attn.py:250]: paddle.nn.Layer

File: examples/ocr/seq2seq_attn.py [hunk 2]
```diff
@@ -322,12 +320,11 @@ def forward(self, inputs, *args):
         return rs
 
 
-class WeightCrossEntropy(Loss):
+class WeightCrossEntropy(fluid.dygraph.Layer):
```
Comment (saxon-zh) [inline examples/ocr/seq2seq_attn.py:323]: paddle.nn.Layer

File: examples/ocr/seq2seq_attn.py (hunk 1 above)
Comment (qingqing01) [reply examples/ocr/seq2seq_attn.py]: Done. Thanks!

File: examples/ocr/seq2seq_attn.py (hunk 2 above)
Comment (qingqing01) [reply examples/ocr/seq2seq_attn.py]: Done. Thanks!

**Review by saxon-zh — APPROVED** [review]
LGTM

**Review by guoshengCS — APPROVED** [review]

**Pull request merged**